build.sbt

```diff
@@ -99,18 +99,26 @@ lazy val `anorm-iteratee` = (project in file("iteratee"))
     ).map("org.specs2" %% _ % "3.8.3" % Test)
   ).dependsOn(anorm)
 
+val akkaVer = "2.4.8"
+
 lazy val `anorm-akka` = (project in file("akka"))
   .enablePlugins(PlayLibrary)
   .settings(scalariformSettings: _*)
   .settings(
     previousArtifacts := Set.empty,
+    resolvers ++= Seq(
+      // For Akka Stream TestKit 'tests' (see akka/akka#21028)
+      "Tatami Releases".at(
+        "https://raw.github.com/cchantep/tatami/master/releases")),
```

Comment:
That's the repository base URL, directory listing is not enabled (and not required for the dependency resolution); e.g. https://raw.github.com/cchantep/tatami/master/releases/com/typesafe/akka/akka-stream-testkit_2.11/2.4.8/akka-stream-testkit_2.11-2.4.8.pom.md5